src/meta/processors/admin/AdminClient.h

```diff
@@ -0,0 +1,70 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef META_PROCESSORS_ADMIN_STORAGEADMINCLIENT_H_
+#define META_PROCESSORS_ADMIN_STORAGEADMINCLIENT_H_
+
+#include "base/Base.h"
+#include <folly/executors/IOThreadPoolExecutor.h>
+#include "base/Status.h"
+#include "thrift/ThriftClientManager.h"
+
+namespace nebula {
+namespace meta {
+
+class FaultInjector {
```

Comment:
What's the point of `Injector` part in the name?  To inject(introduce) a fault?  Kind of misleading, maybe we should come up with a better name?   FaultRecovery/FaultRejector/SafeGuardian etc, WDYT? Correct me if i misunderstood

Reply:
Fault Injector is an component to inject some faults in the UTs.  
It could help us to mock the some corner cases.